src/client.rs

```diff
@@ -8,23 +8,17 @@
 //! # use std::path::PathBuf;
 //! # use std::str::FromStr;
 //! # use tuf::{Result, Tuf};
-//! # use tuf::crypto::KeyId;
+//! # use tuf::crypto::PublicKey;
 //! # use tuf::client::{Client, Config};
 //! # use tuf::metadata::{RootMetadata, Role, MetadataPath, MetadataVersion};
 //! # use tuf::interchange::Json;
 //! # use tuf::repository::{FileSystemRepository, HttpRepositoryBuilder};
 //!
-//! static TRUSTED_ROOT_KEY_IDS: &'static [&str] = &[
-//!     "4750eaf6878740780d6f97b12dbad079fb012bec88c78de2c380add56d3f51db",
-//!     "01892c662c8cd79fab20edec21de1dcb8b75d9353103face7fe086ff5c0098e4",
-//!     "0823260c29956d0b64baa19a64c49c87922ba2bc532d09a6965aef044da490bd",
-//! ];
+//! static TRUSTED_ROOT_KEYS: &'static [PublicKey] = &[];
 //!
 //! # fn main() -> Result<()> {
 //! # block_on(async {
-//! let key_ids: Vec<KeyId> = TRUSTED_ROOT_KEY_IDS.iter()
-//!     .map(|k| KeyId::from_str(k).unwrap())
-//!     .collect();
+//! let keys: Vec<PublicKey> = TRUSTED_ROOT_KEYS.iter().cloned().collect();
```

Comment:
with_trusted_root_keys wants an IntoIterator of &PublicKey, which the slice impls directly, so this logic is no longer necessary.